dispatch/queue.h

```diff
@@ -990,6 +990,54 @@ DISPATCH_EXPORT DISPATCH_PURE DISPATCH_WARN_RESULT DISPATCH_NOTHROW
 void *
 dispatch_get_specific(const void *key);
 
+#if __linux__
```

Comment:
this should NOT be API but private interface in queue_private, and I don't understand why using the _4CF hooks isn't good enough here.